backend/alembic.ini

```diff
@@ -3,7 +3,7 @@
 [alembic]
 # path to migration scripts
 # Use forward slashes (/) also on windows to provide an os agnostic path
-script_location = migrations
+script_location = app/migrations
```

Comment:
is there a reason we needed to move migration scripts into the app?

Reply:
I moved it a while back to figure out the migration issue, but it's good now. Moved it back to migrations folder